### PR TITLE
LPS-177500 | Added an ignore to test and property for a11y tests

### PR DESCRIPTION
**Pull request by andreluis2022**
Background:
Add an ignore and property in the Test, (a11y tests)

JIRA Ticket:
https://issues.liferay.com/browse/LPS-177500

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase [hunk 1]
```diff
@@ -21,6 +22,7 @@ definition {
 	}
 
 	@description = "Verify poshi function AssertAccessible passes atleast one portal page based on WCAG 2.0 AA standards"
+	@ignore = "Test Stub"
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase:25]:
```suggestion
	@ignore = "true"
```

**Review by mateusralv — APPROVED** [review]
Hey @andreluis2022 can you make these minor changes and update the PR?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase (hunk 1 above)
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase:25]: Yeah

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase (hunk 1 above)
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase:25]: Hey @andreluis2022 could you take a look this?

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**